velox/functions/prestosql/Probability.h

```diff
@@ -91,5 +92,29 @@ struct BinomialCDFFunction {
   }
 };
 
+template <typename T>
+struct InverseLaplaceCDFFunction {
+  VELOX_DEFINE_FUNCTION_TYPES(T);
+  FOLLY_ALWAYS_INLINE void
+  call(double& result, double location, double scale, double p) {
+    if (std::isnan(location) || std::isnan(scale) || std::isnan(p)) {
```

Comment:
This code would be easier to read if written as 
if(std::isnan(...) {
  result = ...
  return;
}

if (scale....) {
  result = ....
  return;
}

without the nested ifs.